openacademy/models/1

```diff
@@ -0,0 +1,78 @@
+from odoo import models
+from odoo import fields
+from odoo import api
+from odoo import exceptions
```

Comment:
to many imports 

you can do 

from odoo import models, fields, api, exceptions ........